samples/Playground/Playground/Playground/Playground.Shared/App.xaml

```diff
@@ -10,8 +10,8 @@
                 <XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
 
 				<!-- Load the Material control resources -->
-				<MaterialResources xmlns="using:Uno.Material"
-								   WithImplicitStyles="True" />
+				<MaterialResources xmlns="using:Uno.Material"/>
+				<!--WithImplicitStyles="True" />-->
```

Comment:
Was this commented out for now because of an exception at runtime? Something about setting the Source of a ResourceDictionary? If so, it should be fixed in the latest dev packages from [this commit](https://github.com/unoplatform/Uno.Themes/commit/bc3b542cfcc3e2ab319a12d845575424e245bbd3)

Reply:
Reverted to include WithImplicitStyles.